docs/source/reference/training-guide.rst

```diff
@@ -37,6 +37,27 @@ Example configuration:
       A100-80GB:1  
       H100:1  
 
+Use EFA
+~~~~~~~
+AWS Elastic Fabric Adapter (EFA) is a network interface similar to Nvidia Infiniband that enables users to run applications requiring high levels of inter-node communications at scale on AWS. You can enable EFA on AWS HyperPod/EKS clusters with a simple additional setting in your SkyPilot YAML.
```

Comment:
With the GCP GPU direct merged, could we make this two tabs, one for `AWS EFA` and another for `GCP GPUDirect TCPX`, and each include the instruction for setting the specific networking?

Reply:
Updated